FORMULA/crowdin.rb

```diff
@@ -0,0 +1,16 @@
+class Crowdin < Formula
+  desc "Crowdin-CLI is a command-line tool that allows to manage your resources"
```

Comment:
Reword the description to be `Command-line tool that allows to manage your resources`